pkg/minikube/constants/constants.go

```diff
@@ -436,3 +436,12 @@ const (
 	// KVMDocumentation the documentation of the KVM driver
 	KVMDocumentation = "https://github.com/kubernetes/minikube/blob/master/docs/drivers.md#kvm2-driver"
 )
+
+const (
+	// Windows OS
+	Windows = "windows"
+	// Darwin OS
+	Darwin = "darwin"
+	// Unix OS
+	Unix = "unix"
```

Comment:
Please use Linux here. We don't want to get into the UNIX wars again (and Darwin actually is UNIX, but anyway)